cmd/kubeadm/app/cmd/phases/preflight.go

```diff
@@ -17,23 +17,43 @@ limitations under the License.
 package phases
 
 import (
+	"bytes"
 	"fmt"
+	"html/template"
```

Comment:
I think we want `text/template` here, sorry didn't realize about it in my first pass.

Reply:
This got autoadded by my editor

Reply:
yes, please change to `text/template`.

Reply:
done